src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -952,6 +952,19 @@ PaError OpenStream( struct PaUtilHostApiRepresentation *hostApi,
         goto error;
     }
 
+    /* Allocate memory for source and sink names. */
+    stream->sourceStreamName = (char*)PaUtil_AllocateMemory(strlen("Portaudio source"));
```

Comment:
+1 for '\0'

Reply:
Updated to use sizeof() so +1 for '\0' should already be included.